ledger/src/blockstore_db.rs

```diff
@@ -835,6 +835,11 @@ fn get_cf_options() -> Options {
     options.set_level_zero_file_num_compaction_trigger(file_num_compaction_trigger as i32);
     options.set_max_bytes_for_level_base(total_size_base);
     options.set_target_file_size_base(file_size_base);
+
+    // "By default, RocksDB uses only one background thread for flush and compaction ... You almost
+    // definitely want to call this function if your system is bottlenecked by RocksDB."
+    options.increase_parallelism(3);
```

Comment:
We already set it for the whole db in get_db_options, does it help to do it per-column family?

Reply:
oh we do?  not sure if this helps, I was just trying it out